src/common/error-messages.ts

```diff
@@ -17,4 +17,5 @@ export class ErrorMessages {
   static readonly socialErrorMessage = '로그인에 실패했습니다. 다시 시도해주세요.\n오류가 반복되면 고객센터에 문의해주세요.';
   static readonly tooShortPassword = '최소 글자 수를 채워주세요.';
   static readonly userNotFoundMessage = '회원 정보가 존재하지 않습니다.';
+  static readonly notFoundLiketarget = '좋아요 대상이 존재하지 않습니다.';
```

Comment:
```suggestion
  static readonly notFoundLiketarget = (target: string) => '좋아요할 ${target}을/를 존재하지 않습니다.';
```

usage:
throw Exceptions.notFoundLikeTarget('식당')

Reply:
해당 부분 남겨주신 부분이 더 좋은것 같아요 수정해놓겠습니다~

Reply:
다시보니 `을/를` 이 아니라 `이/가` 가 맞는것 같네요